src/repository.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+class Repository {
```

Comment:
No te olvides de los _includes_:

```suggestion
#pragma once

#include <string>
#include <vector>

#include "file.hpp"
#include "branch.hpp"
#include "branchmanager.hpp"
#include "stashmanager.hpp"

using std::string;
using std::vector;

class Repository {
```